docs/InstallGuide.md

```diff
@@ -131,294 +138,221 @@ To automatically launch the mysql server, execute the following command.
+from one guest to another.
 
-+ vm1, vm3 -> vnet1
-+ vm2, vm4 -> vnet2
+As a representation of the guest here we use [LXC](https://linuxcontainers.org), which helps users run multiple isolated Linux system containers. Any other virtualization technologies might be used, however we take LXC since it is easy to create/destroy/configure and simple enough to do this verification.
 
-The command "run-vm-all" starts all four virtual machines at once.
+Install LXC
 
-    # cd /root/vnet-test-kvm/vm
-    # ./run-vm-all
-    *** vm1 ***
-    node-id: 1
-    interface name: if-x5vjmksl
-    mac address: 52:54:00:0d:84:8c
-    *** vm2 ***
-    node-id: 2
-    interface name: if-sve1s6xp
-    mac address: 52:54:00:9a:cc:00
-    *** vm3 ***
-    node-id: 3
-    interface name: if-bvixz40n
-    mac address: 52:54:00:0a:fb:78
-    *** vm4 ***
-    node-id: 4
-    interface name: if-7st2alal
-    mac address: 52:54:00:e5:f9:fb
+```
+# yum -y install lxc lxc-templates
+```
 
-### Check the IP address of virtual machines
+Create and mount cgroup
 
-You can log into the virtual machines using the telnet command.
+```
+# mkdir /cgroup
+# echo "cgroup /cgroup cgroup defaults 0 0" >> /etc/fstab
+# mount /cgroup
+```
 
-    # telnet localhost PORT-NO
+Create 2 LXC guests
 
-The port numbers are:
+```
+# lxc-create -t centos -n inst1
+# lxc-create -t centos -n inst2
+```
 
-+ vm1 : 5001
-+ vm2 : 5002
-+ vm3 : 5003
-+ vm4 : 5004
+Configure interfaces of each guest
 
-The ip addresses are assigned as follows. As you can see, vm1 and vm3, share the same ip address, as do vm2 and vm4.
+```
+# vi /var/lib/lxc/inst1/config
+lxc.network.type = veth
+lxc.network.flags = up
+lxc.network.veth.pair = inst1
+lxc.network.ipv4 = 10.100.0.10
+lxc.network.hwaddr = 10:54:FF:00:00:01
+lxc.rootfs = /var/lib/lxc/inst1/rootfs
+lxc.include = /usr/share/lxc/config/centos.common.conf
+lxc.arch = x86_64
+lxc.utsname = inst1
+lxc.autodev = 0
+```
 
-+ vm1 : 10.102.0.10
-+ vm2 : 10.102.0.10
-+ vm3 : 10.102.0.11
-+ vm4 : 10.102.0.11
+```
+# vi /var/lib/lxc/inst2/config
+lxc.network.type = veth
+lxc.network.flags = up
+lxc.network.veth.pair = inst2
+lxc.network.ipv4 = 10.100.0.11
+lxc.network.hwaddr = 10:54:FF:00:00:02
+lxc.rootfs = /var/lib/lxc/inst2/rootfs
+lxc.include = /usr/share/lxc/config/centos.common.conf
+lxc.arch = x86_64
+lxc.utsname = inst2
+lxc.autodev = 0
+```
 
-Log into the virtual machines and make sure their ip addresses are correct.
+Make sure that the IPv4 address and MAC address are the same as what you specify when you create the interface database records.
 
-**ex) vm1**
+Launch the LXC guests then enslave the tap interfaces to the datapath.
 
-    # telnet localhost 5001 <- vm1
-    Trying 127.0.0.1...
-    Connected to localhost.
-    Escape character is '^]'.
+```
+# lxc-start -d -n inst1
+# lxc-start -d -n inst2
```

Comment:
LXC failed to start after following this guide on a minimal Centos 6.6 install. I started it again in the foreground while keeping an eye on `/var/log/messages`. I'll investigate what's going on a bit more.

Output from foreground LXC process:

```
lxc-start: conf.c: lxc_mount_auto_mounts: 801 No such file or directory - error mounting proc on /usr/lib64/lxc/rootfs/proc flags 14
lxc-start: conf.c: lxc_setup: 4153 failed to setup the automatic mounts for 'inst1'
lxc-start: start.c: do_start: 688 failed to setup the container
lxc-start: sync.c: __sync_wait: 51 invalid sequence number 1. expected 2
lxc-start: start.c: __lxc_start: 1080 failed to spawn 'inst1'
lxc-start: lxc_start.c: main: 342 The container failed to start.
lxc-start: lxc_start.c: main: 346 Additional information can be obtained by setting the --logfile and --logpriority options.
```

In `/var/log/messages`:

```
Apr 24 15:20:39 joske yum[28210]: Installed: openssh-clients-5.3p1-104.el6_6.1.x86_64
Apr 24 15:20:40 joske yum[28210]: Installed: rsyslog-5.8.10-10.el6_6.x86_64
Apr 24 15:20:40 joske yum[28210]: Installed: 12:dhclient-4.1.1-43.P1.el6.centos.1.x86_64
Apr 24 15:20:40 joske yum[28210]: Installed: passwd-0.77-4.el6_2.2.x86_64
Apr 24 15:20:40 joske yum[28210]: Installed: 2:vim-minimal-7.2.411-1.8.el6.x86_64
Apr 24 15:20:40 joske yum[28210]: Installed: rootfiles-8.1-6.1.el6.noarch
Apr 24 15:21:52 joske yum[28600]: Installed: rsync-3.0.6-12.el6.x86_64
Apr 24 15:29:37 joske kernel: ADDRCONF(NETDEV_UP): inst1: link is not ready
Apr 24 15:29:37 joske kernel: lo: Disabled Privacy Extensions
Apr 24 15:29:38 joske kernel: ADDRCONF(NETDEV_CHANGE): inst1: link becomes ready
Apr 24 15:30:26 joske kernel: ADDRCONF(NETDEV_UP): inst1: link is not ready
Apr 24 15:30:26 joske kernel: lo: Disabled Privacy Extensions
Apr 24 15:30:27 joske kernel: ADDRCONF(NETDEV_CHANGE): inst1: link becomes ready
```